util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py

```diff
@@ -215,15 +217,15 @@ def sanitize_method(method: Optional[str]) -> Optional[str]:
         return method
     return "UNKNOWN"
 
-
-def get_custom_headers(env_var: str) -> List[str]:
-    custom_headers = environ.get(env_var, [])
+def get_custom_headers(env_var: str) -> list[str]:
+    custom_headers = environ.get(env_var, None)
```

Comment:
Why is this change necessary?

Reply:
It avoids allocating a list

Reply:
If I understand this correctly, the list is allocated anyway(in line 228)

Reply:
Ah true, it’s just the pattern I’m used to to avoid the common gotcha.

In any case I believe what was there could return `""` which is why I changed it in the first place.